src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlConsumption.cs

```diff
@@ -702,7 +715,7 @@ private static IReadOnlyList<string> GetFolders(XElement documentElement)
                 return null;
             }
 
-            var folderContainers = folderAttribute.Value.Split(new[] { '\\' }, StringSplitOptions.RemoveEmptyEntries);
+            var folderContainers = folderAttribute.Value.Split(new[] { PathUtilities.DirectorySeparatorChar }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
this worries me.  does this mean our tests behave different on different platforms?

Reply:
Yes. But isn't that what we want when the code we are testing behaves differently on different platforms? Of course, this would requires the testcase to be constructed in such a way that is platform dependent (all test in the PR are created like this).